ModelicaTest/MultiBody.mo

```diff
@@ -2473,9 +2464,9 @@ menu (this sets \"fixed=false\" on parameter \"length\").
 
       extends Modelica.Icons.Example;
       parameter Real tol=1e-4;
-      SI.Force rod_f_diff[3]=rod1.frame_b.f - rod3.frame_b.f
+      SI.Force rod_f_diff[3]=abs(rod1.frame_b.f - rod3.frame_b.f)
```

Comment:
I think I would have preferred to have this line (and the next similar one) unchanged and only have the abs in the assert-clause; it makes it easier to plot the difference.